demos/serverless/deploy.js

```diff
@@ -272,8 +273,8 @@ function ensureRegion() {
 }
 
 function ensureMediaPipelinesRegion() {
-  if (!(new Set(supportedMediaPipelinesControlRegions)).has(region)) {
-      console.error(`Amazon Chime SDK Media Pipelines does not support ${region} (control region). Specify one of the following regions: ${supportedMediaPipelinesControlRegions.join(', ')}.\nSee https://docs.aws.amazon.com/chime-sdk/latest/dg/sdk-available-regions.html#sdk-media-pipelines for more information.`);
+  if (!(new Set(supportedMediaPipelinesControlRegions)).has(mediaPipelinesControlRegion)) {
```

Comment:
Looks like `supportedMediaPipelinesControlRegions ` also needs to be updated?

Reply:
No, Media pipeline is not extended to CPT

Reply:
It's not about CPT, if you check line 58, there are only 4 regions under `supportedMediaPipelinesControlRegions`, but looks like there are more regions according to this [doc](https://docs.aws.amazon.com/chime-sdk/latest/dg/sdk-available-regions.html#sdk-media-pipelines), since we are changing this line, I'm asking if we should also update `supportedMediaPipelinesControlRegions`.

Reply:
Updated, thank you!